FluidNC/src/Maslow/MotorUnit.cpp

```diff
@@ -89,17 +64,6 @@ double MotorUnit::getCurrent(){
     return motor.readCurrent();
 }
 
-/*!
- *  @brief  Computes and returns the error in the axis positioning
- */
-double MotorUnit::getError(){
```

Comment:
I know that we aren't calling this function, but I think we will want to use this eventually (but we can put it back in). I think if any arm has too large of an error (maybe more than 1mm?) we should throw an error and stop because probably something is wrong.